SlideMenu/Source/SlideNavigationController.m

```diff
@@ -445,7 +445,8 @@ - (UIBarButtonItem *)barButtonItemForMenu:(Menu)menu
 	}
 	else
 	{
-		UIImage *image = [UIImage imageNamed:MENU_IMAGE];
+        NSBundle* bundle = [NSBundle bundleForClass:[SlideNavigationController class]];
```

Comment:
Thanks for the PR.
Could you please update the code to match the same indentation as the rest of the code, also with tabs instead of spaces. Thanks

:shipit: after that